style.md

```diff
@@ -13,9 +13,12 @@ The redundancy reduction from removing the namespacing prefix is not useful enou
 ## Optional settings should have pointer Go types
 
 So we have a consistent way to identify unset values ([source][optional-pointer]).
+The exceptions are slices, maps, and booleans where the default should be false, in which case `omitempty` is sufficient and no pointer is needed ([source][no-pointer-for-slices] and [source][no-pointer-for-boolean]).
```

Comment:
> But sometimes false does mean “don't change” (and those are the
booleans where the default should be false)

That'll be a problem if we want some `bool` values to be updated (live update), we surely want to change some booleans from `true` to `false`.